addons/portal/views/portal_templates.xml

```diff
@@ -200,7 +202,7 @@
         :param classes : navbar classes
     -->
     <template id="portal_searchbar" name="Portal Search Bar">
-        <nav t-attf-class="navbar navbar-light navbar-expand-lg border py-0 o_portal_navbar {{classes if classes else ''}} {{'mt-4 rounded-top border-bottom-0' if breadcrumbs_searchbar else 'border-top-0 mb-2'}}">
+        <nav t-attf-class="navbar navbar-light navbar-expand-lg border py-0 mb-2 o_portal_navbar {{classes if classes else ''}} {{'mt-3 bg-white rounded' if breadcrumbs_searchbar else 'border-top-0' }}">
```

Comment:
Sorry. Why adding `bg-white` sometimes if it is now always forced to white with your last commit? :)

Reply:
Sorry, you're right, I have missed this guy :) 
Is this the only remark?

Reply:
Yep, fix this and I will merge... normally :)

Reply:
done, it's perfect now (normally ^^)